experimental/plugin.go

```diff
@@ -0,0 +1,30 @@
+package experimental
+
+import (
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// PluginExe is the singleton container for your plugin.
+type PluginExe interface {
+	// CheckHostHealth for the plugin executable
```

Comment:
The doc comment should start with the name of the method - does the following make sense?
```suggestion
	// CheckExeHealth corresponds to CheckHostHealth for the plugin executable.
```